gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/experimental/dsl/ExperimentalUiKitApplication.kt

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2020-2022 JetBrains s.r.o. and respective authors and developers.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.compose.experimental.dsl
+
+import javax.inject.Inject
```

Comment:
Why we need `@Inject` here?

Reply:
So Gradle can inject `name` constructor argument